src/Extensions/TimedScopes/TimedScopeEventSource.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics.Tracing;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Abstractions;
+using Microsoft.Omex.Extensions.Logging.TimedScopes;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	[EventSource(Name = "Microsoft-OMEX-TimedScopes")]
+	internal sealed class TimedScopeEventSource : EventSource
```

Comment:
>internal [](start = 1, length = 8)

nit: comments and spacing between variables

Reply:
Fixed, thanks